### ISSUE TITLE
How do I reproduce MAE performance for sha shb and qnrf datasets

### ISSUE THREAD

**Issue by msn199959** [dk-liang/CLTR#28]
How do I set the number of decoder layer? And hyperparameters like learning rate?

**Comment by kexiaolong0121** [comment]
你好，请问有这两个数据集的解决方案了没有
